index.js

```diff
@@ -180,9 +184,9 @@ function BlinkDiff (options) {
 	}
 
 	this._blockOut = load(options.blockOut, []);
-	if (typeof this._blockOut != 'object' && (this._blockOut.length !== undefined)) {
+	//if (typeof this._blockOut != 'object' && (this._blockOut.length !== undefined)) {
```

Comment:
So, what's going on here? Why did you need to comment out this check? What is the type of this._blockOut? Is it an integer?

Reply:
Oh, do you expect this._blockOut to be an array?